packages/pds/src/app-view/services/feed/views.ts

```diff
@@ -223,6 +223,7 @@ export class FeedViews {
         img.image.ref,
       ),
       alt: img.alt,
+      ...(img.aspectRatio && { aspectRatio: img.aspectRatio }),
```

Comment:
This could probably done more cleanly with a ternary operator or something? I'm bad at JS...

Reply:
I think that’s probably the cleanest way to do it in one line

Reply:
In fact, would a simple `aspectRatio: img.aspectRatio,` do the right thing? Looking at the other parts of the same source file, I think an `undefined` value results in the field being omitted from the final record output.

Reply:
I’d have to check the code but the JSON stringifier removes undefined fields so yeah it would almost certainly be fine. The only issue typically is if you do something like `“aspectRatio” in object` but I highly doubt that’s the case here

Reply:
yeah i think just `img.aspectRatio` would be fine here